### PR TITLE
[LeetCode] increase pull problem node runner timeout

### PR DESCRIPTION
**Pull request by shisoft**

### CODE REVIEW

**Review by yukozh — APPROVED** [review]

File: src/JoyOI.VirtualJudge.LeetCode/LeetcodePullProblemBodyActor.cs [hunk 1]
```diff
@@ -81,6 +81,7 @@ public static void Main()
                 Source = "LeetCode",
                 OriginUrl = baseUrl + problemUri,
                 Templates = templates,
+                TemplatesError = File.ReadAllText("stderr.txt"),
```
Comment (yukozh) [inline src/JoyOI.VirtualJudge.LeetCode/LeetcodePullProblemBodyActor.cs]: Please detect the runner exit code, if exit code is non-zero, please throw the exception, please don't return the body.

**Review by yukozh — CHANGES_REQUESTED** [review]

File: src/JoyOI.VirtualJudge.LeetCode/LeetcodePullProblemBodyActor.cs (hunk 1 above)
Comment (yukozh) [reply src/JoyOI.VirtualJudge.LeetCode/LeetcodePullProblemBodyActor.cs]: Also, we don't have the field which named TemplatesError and Templates, please take a look into ProblemJson class in OnlineJudge repo.

File: src/JoyOI.VirtualJudge.LeetCode/LeetcodePullProblemBodyActor.cs [hunk 2]
```diff
@@ -8,46 +8,52 @@
...
-                {
-                    File.WriteAllText("error.txt", ex.ToString());
-                    File.WriteAllText("return.json", JsonConvert.SerializeObject(new { Outputs = new[] { "error.txt" } }));
-                }
-                else
-                {
-                    await Task.Delay(3000);
-                    goto main;
-                }
+            var retryLeftTimes = 3;
```
Comment (yukozh) [inline src/JoyOI.VirtualJudge.LeetCode/LeetcodePullProblemBodyActor.cs:36]: What's the differences between the old and new?

**Comment by shisoft** [comment]
Nothing, I think this is just formats

On 15 Nov 2017, 7:39 AM +0800, Yuko Zheng <notifications@github.com>, wrote:

@yukozh commented on this pull request.

________________________________

In src/JoyOI.VirtualJudge.LeetCode/LeetcodePullProblemBodyActor.cs<https://github.com/JoyOI/VirtualJudge/pull/7#discussion_r150996796>:

> -                File.WriteAllText("return.json", JsonConvert.SerializeObject(new { Outputs = returnFiles }));
-            }
-            catch (Exception ex)
-            {
-                --retryLeftTimes;
-                if (retryLeftTimes <= 0)
-                {
-                    File.WriteAllText("error.txt", ex.ToString());
-                    File.WriteAllText("return.json", JsonConvert.SerializeObject(new { Outputs = new[] { "error.txt" } }));
-                }
-                else
-                {
-                    await Task.Delay(3000);
-                    goto main;
-                }
+            var retryLeftTimes = 3;


What's the differences between the old and new?

**Comment by yukozh** [comment]
You still did not resolved the conflicts.

**Comment by shisoft** [comment]
cuz I don't know how to do it on github. I can't make changes on this repo and the resolve button just don't work here

**Pull request merged**